src/app/mod.rs

```diff
@@ -739,7 +739,12 @@ impl App {
                 },
             )
             .await?;
-        Ok(id)
+        let expected_time = if let Some(interval) = payout_queue.spawn_in() {
+            job::next_attempt_of_queue(&self.pool, payout_queue.id, interval).await?
```

Comment:
I would fall back to now + interval here. We don't want this command to fail just because this query fails. Its optional.